pkg/backend/httpstate/stack.go

```diff
@@ -119,13 +122,17 @@ func newStack(apistack apitype.Stack, b *cloudBackend) Stack {
 	stackName, err := tokens.ParseStackName(apistack.StackName.String())
 	contract.AssertNoErrorf(err, "unexpected invalid stack name: %v", apistack.StackName)
 
+	defaultOrg, err := backend.GetDefaultOrg(context.TODO(), b, b.currentProject)
+	contract.AssertNoErrorf(err, "unable to look up default org")
```

Comment:
Couldn't this fail for a user if the network hiccuped? I think this needs to actually bubble up an error result